tests/settings.py

```diff
@@ -15,7 +15,8 @@
 
 SECRET_KEY = "testing"
 
-DEBUG = "INTERACTIVE" in os.environ
+# Override in `local.py`
+DEBUG = False
```

Comment:
Thanks for reverting the cache settings @katdom13, could you also revert this line to the original `DEBUG = "INTERACTIVE" in os.environ`?
Because I'm using `tox -e interactive` instead of calling `manage.py runserver`, which is the reason for this env-variable.

Everything else looks good to me, thanks again! I'm back in office on monday, I will try to merge and release it next week.

Reply:
@th3hamm0r 
I see. I reverted the variable change now.
Thanks!